modules/etcd-certificates.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificate-types-descriptions.adoc
+
+[id="etcd-certificates_{context}"]
+= etcd certificates
+
+[discrete]
+== Purpose
+
+etcd certificates are signed by the etcd-signer; they come from a certificate
+authority (CA) that is generated by the bootstrap process.
+
+[discrete]
+== Location
+
+* CA certificates:
+** etcd CA certificate: `/etc/ssl/etcd/ca.crt`
+** etcd metric CA certificate: `/etc/ssl/etcd/metric-ca.crt`
+* Server certificates: `/etc/ssl/etcd/system:etcd-server`
+* Client certificates: `<api_server_pod_directory>/secrets/etcd-client/`
+* Peer certificates: `/etc/ssl/etcd/system:etcd-peer`
+* Metric certificates: `/etc/ssl/etcd/metric-signer`
+
+[discrete]
+== Expiration
+
+The CA certificates are valid for 10 years. The peer, client, and server
+certificates are valid for three years.
+
+[discrete]
+== Management
+
+These certificates are managed by the system and not the user.
+
+[discrete]
+== Services
+
+etcd certificates are used for encrypted communication between etcd member
+peers, as well as encrypted client traffic. The following certificates are
+generated and used by etcd and other processes that communicate with etcd:
+
+* Peer certificates: Used for communication between etcd members.
+* Client certificates: Used for encrypted server-client communication. Client
+certificates are currently used by the API server only, and no other service
+should connect to etcd directly except for the proxy.
+* Server certificates: Used by the etcd server for authenticating client requests.
+* Metric certificates: All metric consumers connect to proxy with metric-client
+certificates.
```

Comment:
Is this doc for ocp 4.3 only or for 4.4? if it only for 4.3, it should be ok, regarding to 4.4, there is not dir: /etc/ssl/etcd directory. 
sh-4.2# cd /etc/ssl
sh-4.2# ls
certs
sh-4.2# cd certs/
sh-4.2# ls
ca-bundle.crt  ca-bundle.trust.crt